examples/credential_metadata_sd_jwt_vc.json

```
@@ -12,11 +12,22 @@
       "display": [
         {
           "name": "IdentityCredential",
+          "logo": {
+                    "uri": "https://university.example.edu/public/logo.png",
+                    "alt_text": "a square logo of a university"
+                  },
```

Trivial fix of indentation:

```suggestion
          },
```